lib/rest/accounts/v1/credential/aws.ts

```diff
@@ -181,9 +184,14 @@ export class AwsContextImpl implements AwsContext {
     return operationPromise;
   }
 
-  update(params?: any, callback?: any): Promise<AwsInstance> {
+  update(
+    params?:
+      | AwsContextUpdateOptions
+      | ((error: Error | null, item?: AwsInstance) => any),
+    callback?: (error: Error | null, item?: AwsInstance) => any
+  ): Promise<AwsInstance> {
     if (typeof params === "function") {
-      callback = params;
+      callback = params as (error: Error | null, item?: AwsInstance) => any;
```

Comment:
Is this needed? Not seeing any TS warnings w/o it.

Reply:
It was previously needed for [flowRevision.ts](https://github.com/twilio/twilio-node/actions/runs/3802537470) but the error is now gone. Will removed it